wasi-nn.witx

```diff
@@ -68,6 +87,12 @@
     (param $tensor $tensor)
     (result $error (expected (error $nn_errno)))
   )
+  (@interface func (export "set_input_by_name")
+      (param $context $graph_execution_context)
+      (param $name string)
+      (param $tensor $tensor)
+      (result $error (expected (error $nn_errno)))
+    )
```

Comment:
This isn't necessary for named models; I don't think it's necessary at all, but if there's a pressing need I don't see we could discuss it in a separate PR?